dimod/sampleset.py

```diff
@@ -453,6 +453,27 @@ def infer_vartype(samples_like):
 
     raise ValueError("given samples_like is of an unknown vartype")
 
+def drop_variables(sampleset, variables_to_drop):
+    """Drop variables from a sampleset
+
+    Args:
+        sampleset (:obj:`.SampleSet`):
+            Original sampleset.
+
+        variables_to_drop (iter):
```

Comment:
should be `container`, not `iter`. An iterator won't actually work in this context.